R/merge_hydrofabrics.R

```diff
@@ -51,12 +52,31 @@ build_new_id_table = function(x,
     st_drop_geometry() %>%
     renamer()
   
-  if("set" %in% names(fl)) {
-    fl <- select(fl, oldID = id, set)
-  } else {
-    fl = select(fl, oldID = id, member_comid)
-  }
+  # need to get member_comid for each flowpath out of the lookup table
+  if(!"member_comid" %in% names(fl)) {
+    lu <- read_sf(x$path, lookup_table_layer) %>% 
+      renamer() 
+    
+    names(lu) <- tolower(names(lu))
```

Comment:
I have been wondering this - shouldn't the names in all lookups be lowercase anyway to match our other tables?

Reply:
Yes -- I think this is what we should be aiming for. Some kind of flexibility, for now at least, is needed though.